packages/palette/src/shared/RequiredField.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import { Text, TextProps } from "../elements"
+
+export const RequiredField: React.FC<TextProps> = (props) => {
+  return (
+    <Text variant="xs" color="black60" textAlign="left" {...props}>
+      *Required
+    </Text>
```

Comment:
This is not how we denote required fields in the design system, and I don't really buy the reasoning for this being a component.

Reply:
Oh, duh! Sorry, I totally forgot the context of this PR!

Reply:
(Of course, feel free to leave it if that's universally how we denote required state now)